data_steward/bq_utils.py

```diff
@@ -61,6 +61,14 @@ def get_ehr_rdr_dataset_id():
     return os.environ.get('EHR_RDR_DATASET_ID')
 
 
+def get_ehr_rdr_deid_dataset_id():
```

Comment:
The simple answer to your question is no.  `get_combined_deid_dataset_id()` is used by `curation/data_steward/cdr_cleaner/clean_cdr.py`.  

Both functions are attempting to get the environment value for `EHR_RDR_DEID_DATASET_ID`.  The difference is what happens if the value isn't set in the environment.  
`get_ehr_rdr_deid_dataset_id` will return null.
`get_combined_deid_dataset_id` will return a dataset string with the current date formatted as part of the name.  

So, which option is preferred?  And which name?  And why not defined these constants in the `curation/data_steward/constants/bq_utils.py` file?  This will make the names easily accessible to test instances if they are needed.

Reply:
We should consolidate somewhere.

Reply:
@lrwb-aou yes, you're right, `get_combined_deid_dataset_id()` **is** used by clean_cdr! Sorry, I'm not sure how I missed this. I would like to make adding of the cleaning rules to the pipeline be a separate task from changing dataset naming conventions. Please share if you have any ideas on how we could consolidate these or if you think we should defer and keep this as-is for now.

I also agree we should store these constants in one place. @ksdkalluri let me know if you'd like me to take care of all the bq constants in a separate pull request so you can focus on this one.

Reply:
@mark-velez .  I'm not suggesting we change dataset naming conventions here.  That function was added so we can run cleaning rules on the correct combined_deid dataset.  Both functions are reading the same environment variable.  The consolidation areas are:
1.  What to do if the environment variable doesn't exist (return null or try to make the best reasonable guess and return that)?
2.  Which name is a better/more descriptive name for the function to use?  `get_ehr_rdr_deid_dataset_id` or `get_combined_deid_dataset_id`.  I vote for the `get_combined_deid_dataset_id` because we have a dataset named `combined<insert_date>_deid`.

Reply:
Thanks @lrwb-aou for breaking it down to two points. I only bring up dataset naming because **I** may mention it and I want to be clear that we do not want that to be in this PR's scope.
1. I think if callers require the dataset name and it hasn't been properly configured then just failing may be more appropriate than trying to guess. @lrwb-aou @ksdkalluri please let me know what you think. If _we are_ going to guess here then perhaps we should be consistent and do the same guessing with other unprovided dataset names. If providing the name is optional because it can be inferred, then I think we should consider eliminating the explicit parameter altogether. With a more robust naming convention it may be possible that correct dataset names can be inferred and we wouldn't have to configure so much.
2. I also vote for `get_combined_deid_dataset_id` rather than `get_ehr_rdr_deid_dataset_id`.

Reply:
@ksdkalluri as discussed I was expecting only for this variable's name to be changed rather than all that have ehr_rdr, but after perusing the changes I think it is ok.